frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/import-providers-tab/providers/vm-import-provider-controller-errors.tsx

```diff
@@ -14,6 +14,40 @@ import { errorsFirstSort } from '../../../../../k8s/enhancedK8sMethods/k8sMethod
 import { ResultTabRow } from '../../result-tab/result-tab-row';
 import { resultContentToString } from '../../../utils/utils';
 import { iGetImportProviders } from '../../../selectors/immutable/import-providers';
+import { ExternalLink } from '@console/internal/components/utils';
+import {
+  VMWARE_DOCURL,
+  OVIRT_DOCURL,
+  V2VProviderErrorSpecialUIMessageRequest,
+} from '../../../../../constants/v2v';
+
+const getDocURL = (providerType: VMImportProvider) =>
+  providerType === VMImportProvider.VMWARE ? VMWARE_DOCURL : OVIRT_DOCURL;
+
+const resolveUIMessageTemplating = (message: string, provider: VMImportProvider) => {
+  if (!message) {
```

Comment:
if message is `null` or `undefined` ?

Reply:
that shouldn't be an issue as it will just pass through any empty value without a change

Reply:
ok